src/plugins/graph/imports/components/axis/components/axis-or-legend-attribute-menu.tsx

```diff
@@ -70,16 +74,11 @@ const _AxisOrLegendAttributeMenu = ({ place, target, portal,
     });
   }, [attribute?.name, data?.attributes, dataConfig, labelText, setLabelText, attrId]);
 
-  const updateTargetMenuClasses = (isOpen: boolean) => {
-    target?.classList.toggle("target-open", isOpen);
-    target?.classList.toggle("target-closed", !isOpen);
-  };
-
   return (
     <div className={`axis-legend-attribute-menu ${place}`} ref={menuRef}>
       <Menu boundary="scrollParent">
         {({ onClose, isOpen }) => {
-          updateTargetMenuClasses(isOpen);
+          onOpenClose && onOpenClose(isOpen!);
```

Comment:
the `!` after isOpen seems like it ought to be unnecessary based on what the code looked like before. Did you get a type error without it?